beyond-essence/inventory/dropdowns.md

```diff
@@ -0,0 +1,88 @@
+---
+layout: beyond-essence
+header_image: private/main.jpg
+title: Dropdowns
+background_image: index.jpg
+---
+
+Dropdowns belong to the interaction controls.
+They present a list of fixed options the user can choose from.
+
+## Use cases
+
+In the Beyond cockpit we provide two types of dropdowns:
+
+* Dropdowns with default selection. Are used to already preselect the most common or the recommended option for the user.
+
+* Dropdowns with filtered suggestions. Are used to present a large number of options to the user. By filtering the suggestions it's easier and faster for the user to make their decision, or find the required entry.
+
+## Structure
+
+Dropdowns consist of a container, a label, and a drawer.
+The label is located near the top of the container.
+The drawer is displayed above or below the container once the dropdown is selected.
+Dropdowns can vary in width.
+
+Dropdowns with default selection additionally include arrows on the very right of the container.
+
+Exception: Some dropdowns, e.g the pagination dropdown, do not contain a label.
+
+{% image_custom image="/assets/img/pages/essence/dropdowns-pagination.png" width="50" %}
+
+## Position
+
+Dropdowns are positioned on a card and can be arranged with further dropdowns or other UI elements such as checkboxes or input fields.
+
+## Behavior
+
+The two types of dropdowns behave differently.
+
+### Dropdowns with default selection
+
+In the initial state, the container displays the default selection.
+After the container or the arrows are clicked or touched, the drawer is opened above the container and the label.
+The user can now choose between the different dropdown options.
+The currently selected option is marked with a hook on the left.
+Once the user selected an option, the drawer is closed and the selected option is displayed in the container.
+It is not possible to selected more than one option at the same time.
+The drawer can be closed at any time by clicking or touching outside the drawer.
+
+{% image_custom image="/assets/img/pages/essence/dropdown-default-selection.gif"  width="50" %}
+
+### Dropdowns with filtered suggestions
+
```

Comment:
I discussed this topic with Anne und Katja and we all share the same opinion that it is not a dropdown in the classical sense. So we suggest to move this section to the input field topic as an ` input field with preselected proposals`. And keep of course the already existing one `Input fields with suggestions`

Reply:
@BirgitBader Could you adapt the input article accordingly? :) I'll then delete it in this article.